src/components/vaults/VaultSelect.tsx

```diff
@@ -48,7 +48,7 @@ export default function VaultSelect({
         {...textFieldProps}
         onChange={handleChange}>
         <MenuItem value="" disabled>
-          {t('fields.' + name)}
+          {t(label)}
```

Comment:
Looks like this solution contradicts a pattern that we use all over the system.

In many places we add the key `fields` to the language json so we can define the names of fields in a single location.

Looks like in this specific place we don't have the correct key in the language file so it shows the actual key.

I'd like to see a solution that complies with the widely used approach in the code. Indeed it's not written anywhere so we can correct that and add a hint so new devs are not confused.

We've got this code, but it's not wrapped in `fields` object:

https://github.com/podkrepi-bg/frontend/blob/7411045c290e6f70d3b8254ae3f3e38623db690f/public/locales/bg/recurring-donation.json#L12

We need to define the following `fields` in order to have `<VaultSelect />` component working correctly:

```json
"fields": {
  "vaultId": "ID на трезор",
  "sourceVaultId": "От трезор",
  "targetVaultId": "Към трезор"
}
```

Reply:
We also need to add the translation domain to the hook

```tsx
const { t } = useTranslation('vaults')
```

and load it in the concerned pages

```
export const getServerSideProps = securedAdminProps(
  ['common', 'auth', 'vaults', 'admin', 'validation'],
  () => endpoints.vaults.vaultsList.url,
)```